stack.yaml.lock

```diff
@@ -4,9 +4,4 @@
 #   https://docs.haskellstack.org/en/stable/lock_files
 
 packages: []
-snapshots:
-- completed:
-    size: 523700
-    url: https://raw.githubusercontent.com/commercialhaskell/stackage-snapshots/master/lts/14/7.yaml
-    sha256: 8e3f3c894be74d71fa4bf085e0a8baae7e4d7622d07ea31a52736b80f8b9bb1a
-  original: lts-14.7
+snapshots: []
```

Comment:
I think this file can be removed entirely when using nix 👍

Reply:
YES WE CAN!